Makefile

```diff
@@ -10,7 +10,7 @@ endif
 BINARY=keymaster
 
 # These are the values we want to pass for Version and BuildTime
-VERSION?=1.16.0
+VERSION?=1.16.1
```

Comment:
This is a feature release, right?
So the version should be `1.70.0`?
We're doing semantic versioning, I thought?

Reply:
kind of ecdsa support is still not fully backed (as in no combination of certs), and the  ca stuff I consider a bugfix.